src/pages/components/contained-list/usage.mdx

```diff
@@ -124,10 +124,15 @@ and optional interactive elements.
 </Row>
 
 1. **List header area:** Area to group the succeeding list items.
-2. **List title:** Descriptive name for the group of list items.
-3. **List item:** The content with in a list item row.
-4. **Interactive element** (optional): An interactive element in the header or
-   row such as a button, link, icon, or toggle.
+2. **List item area:** Area to group text and elements in a list row.
+3. **List title:** Descriptive name for the group of list items.
+4. **List item:** The content in a list item row.
+5. **Non-interactive icon (optional):** A decorative icon to help visually
+   describe a list item.
+6. **Search icon (optional):** An icon to represent search or filtering
```

Comment:
Same comment for the image, should we call #6 and #7 icons or icon buttons?

Reply:
I can change #6 to reference a button, but #7 does not always need to be a button, it is any kind of interactive element.